_data/simple-icons.json

```diff
@@ -210,6 +235,11 @@
             "hex": "0078D7",
             "source": "http://azure.com/devops"
         },
+        {
+            "title": "Babel",
+            "hex": "F5D955",
```

Comment:
The [official SVG](https://github.com/babel/logo/blob/master/babel.svg) seems to use `#F9DC3E`